### ISSUE TITLE
Is KB5006746 the correct update?

### ISSUE THREAD

**Issue by yamauchikazu** [MicrosoftDocs/windowsserverdocs#7399]
KB5006746 (Os build 22000.282) is cumulative update for Windows 11 ver 21H2. Not for Windows Server 2019.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: bfb84ea4-a463-b510-d87f-c24d625c3d58
* Version Independent ID: 120c0010-b9a1-d60d-d56f-b57cf43d5a06
* Content: [Storage Migration Service frequently asked questions (FAQ)](https://learn.microsoft.com/en-us/windows-server/storage/storage-migration-service/faq)
* Content Source: [WindowsServerDocs/storage/storage-migration-service/faq.yml](https://github.com/MicrosoftDocs/windowsserverdocs/blob/main/WindowsServerDocs/storage/storage-migration-service/faq.yml)
* Product: **windows-server**
* GitHub Login: @nedpyle
* Microsoft Alias: **nedpyle**

**Comment by wbsmolen** [comment]
@yamauchikazu thanks for posting this. Fixed: https://github.com/MicrosoftDocs/windowsserverdocs/commit/793e2c24048f14035ef54fe16bb95d310a81e655

**Comment by yamauchikazu** [comment]
Are you still wrong?
KB5005101 update is for Windows 10 ver 2004/20H2/21H1(os build 19041/19042/19043). Not for Windows Sever 2019 (os build 17763).

**Comment by wbsmolen** [comment]
> Are you still wrong? KB5005101 update is for Windows 10 ver 2004/20H2/21H1(os build 19041/19042/19043). Not for Windows Sever 2019 (os build 17763).

You're correct - my mistake. The *actual* answer is [KB5015880](https://support.microsoft.com/en-us/topic/july-21-2022-kb5015880-os-build-17763-3232-preview-1c984723-cdf0-4a24-9a4f-5df11d3024a1) and I'm updating this now. So sorry!

**Comment by robinharwood** [comment]
We hugely appreciate your contribution, feedback, and support to our technical documentation.

We’ve phased out GitHub Issues on this repository as the feedback mechanism for content and have replaced it with a new feedback system. For more information see, [Content User Feedback](https://aka.ms/ContentUserFeedback). As part of this rollout, we are migrating your issue where our team will use your comments to improve the documentation experience. Unfortunately, this does mean you won't be able to access this issue after the migration is complete.

All issues will be migrated by Thursday 29th August 2024.

Once again, thank you for your valuable feedback, we hope you’ll continue to contribute using our new feedback system.
